data_cleaning/data_cleaning.md

```diff
@@ -0,0 +1,20 @@
+# ROSbot Data Cleaning
+- Once you're done collecting the data, you will need to clean it to remove unwanted parts. 
+- For example, the robot is not moving, or some random guy walks in front of it. 
+- To do this, you can create your own script (or use the one provided). 
+
+## Locate your data directory
+- You will need to find the directory you have stored your data in. 
+- For starting, I recommend that you make a copy of your data somewhere else and attempt to clean that. 
+- Retrieve the directory that all your datacollection folders are in and copy it, you will need to enter that into the command later. 
```

Comment:
This instruction is confusing. Retrieve the directory before copying it? Enter what into what command?
Be more explicit in your instructions. Give an example of how to copy it using terminal commands if that is what you mean by "enter that into the command later". It is unclear and will lead to mistakes when someone tries to follow these instructions as they are written now.